profiles/models.py

```diff
@@ -98,4 +99,7 @@ class Profile(models.Model):
         null=True,
     )
     mailing_address = models.TextField(blank=True, null=True)
-    date_joined = models.DateTimeField(blank=True, null=True)
+    date_joined_micromasters = models.DateTimeField(blank=True, null=True, default=datetime.now)
```

Comment:
I chose `default=datetime.now` instead of `auto_now_add=True` because the latter makes the field read only. Not sure it matters...